src/inference/mh.jl

```diff
@@ -188,6 +188,22 @@ function MH(space...)
     return MH{tuple(syms...), typeof(proposals)}(proposals)
 end
 
+function StaticMH(model::Model)
```

Comment:
It seems this shadows `AMH.StaticMH` (and the same below for `RWMH`)? This can't really be fixed by qualifying these constructors since then it would be type piracy - neither the function nor any of its arguments are owned by Turing.

More generally, my understanding from the PR in AMH was that these functions will be deprecated and renamed as they misleadingly suggest that something of type `StaticMH` will be constructed here.

Reply:
Ok agreed, let's not make the confusion bigger in this PR. 
The user can pass the type of proposal for the priors with a kwarg.